packages/grid/x-grid-modules/src/hooks/root/useEvents.ts

```diff
@@ -213,7 +213,7 @@ export function useEvents(
       const api = apiRef.current!;
 
       return () => {
-        logger.warn('Clearing all events listeners');
+        logger.debug('Clearing all events listeners');
```

Comment:
info might be appropriate here as it's a pretty important step, don't you think?

Reply:
As you prefer, these logs are meant for you in the first place so whatever you are more confortable with. I would personally expect cleanup to be normal, not requiring a lot of attention. Maybe you are more interested in the unmount event? Should, we update #146 to be info too?

Reply:
Yes I think we could reword a bit, we are missing the lifecycle step in the phrase.
Agree to update #146 too, also, `initialising` is missing in #146

Reply:
- initializing US
- initialising UK

Reply:
I have left debug as we use debug in 3 other places for the event listener, feel free to do a second iteration, for now, I focus on consistency. This is master:

https://github.com/mui-org/material-ui-x/blob/9c6c1d7a3d363979bcbd8234ec160946e8bfd0ee/packages/grid/x-grid-modules/src/hooks/root/useEvents.ts#L200
https://github.com/mui-org/material-ui-x/blob/9c6c1d7a3d363979bcbd8234ec160946e8bfd0ee/packages/grid/x-grid-modules/src/hooks/root/useApi.ts#L24
d8234ec160946e8bfd0ee/packages/grid/x-grid-modules/src/hooks/root/useApi.ts#L28